README.md

```diff
@@ -1 +1,46 @@
-# Haal-Centraal-BRP-historie-bevragen
\ No newline at end of file
+# Haal-Centraal-BRP-historie-bevragen
+
+![lint oas](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/lint-oas/badge.svg)
+![generate sdks](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/generate-sdks/badge.svg)
+![generate postman collection](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/workflows/generate-postman-collection/badge.svg)
+
+BRP Historie Bevragen is een Haal Centraal API voor het zoeken en raadplegen van historiesche gegevens met betrekking op een ingeschreven natuurlijke personen voor alle binnengemeentelijke afnemers in NL. De informatie die de API levert is herleidbaar naar het LO GBA 3.10.
+
+# Planning
+Op dit ogenblik wordt er gewerkt aan release v1.0.0. Release v1.0.0 is uitgebracht en is [hier](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v1.0.0) te vinden.
+
+https://eu-rm.roadmunk.com/publish/29a13c572a0dfc79f1d0386fc572e15383afe11f
+
+## Getting started
+Om te beginnen met ontwikkelen van de API is de [getting started](./docs/getting-started.md) een goed beginpunt. De API is technisch gespecificeerd in Open API specificaties (zie hieronder bij documentatie).
+
+## Documentatie
+* [Technische specificaties van de **resolved** versie](./specificatie/genereervariant) (Open API Specificaties), [Technische specificaties van de **unresolved** versie](./specificatie) (Open API versie)
+* Specificaties van de actuele productie versie in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-Historie-bevragen/master/specificatie/genereervariant/openapi.yaml)
+
+* Specificaties van de develop versie in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-Historie-bevragen/develop/specificatie/genereervariant/openapi.yaml)
+
+* Ontwerpkeuzes staan in het document [Design decisions](./docs/design_decisions.md)
+* [Productvisie](./docs/productvision.md)
+* [Definition of Ready](./docs/definition_of_ready.md)
+* [Definition of Done](./docs/definition_of_done.md)
+* Hoe u kunt [bijdragen](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/CONTRIBUTING.md)
+* [Omgangsvormen](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/CODE_OF_CONDUCT.md)
+* [Functionele specificaties](./features)
+
+Naast deze API is er nog een aantal andere aan BRP gerelateerde API's beschikbaar:
+* [Historische BRP-gegevens bevragen](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen)
```

Comment:
Dat is deze API. Moet hier niet de verwijzing naar actueel bevragen?

Reply:
Klopt, aangepast